messages/pt.json

```diff
@@ -1,8 +1,9 @@
 {
-  "editor.base-store-component.title": "IO Base App",
-  "editor.base-store-component.description": "Um componente de demonstração para apps do VTEX IO",
-  "editor.base-store-component.title.title": "Título",
-  "editor.base-store-component.title.description": "Especifique um título personalizado",
-  "base.title": "Olá, Mundo!!",
-  "base.change-value": "Troque seu valor: {value}"
-}
\ No newline at end of file
+  "admin/editor.locale-switcher.title": "Localidade disponível",
+  "admin/editor.locale-switcher.display.title": "Modo de exibição",
+  "admin/editor.locale-switcher.display.description": "Você deveria usar essa propriedade para esconder a opção atual selecionada.",
```

Comment:
I don't think "Você deveria usar" is a good way to express this, since it sounds like an order.
Maybe just `"Esconder opção atual"` would be better